src/components/Review.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom";
+import { getReviewById } from "../utils/api";
+
+export default function Review() {
+    const [review, setReview] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+    const { review_id } = useParams()
+
+    useEffect(() => {
+        setIsLoading(true)
```

Comment:
You don't need this here. The only way to change the review_id is to cause a rerender, which will set your isLoading back to false 😄